metadata_annotation.ipynb

```diff
@@ -0,0 +1,2166 @@
+{
```

Comment:
<p>So based on your comments, we should ensure that the other function has been called before, right? We should enforce this with code -&gt; check if the right information is there and raise an Error if not. The error message should indicate what needs to be run before</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/pertpy-tutorials/pull/16/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>